package.json

```diff
@@ -8,10 +8,10 @@
 		"url": "https://github.com/joyent/node-jsprim.git"
 	},
 	"dependencies": {
-		"assert-plus": "1.0.0",
-		"extsprintf": "1.3.0",
-		"json-schema": "0.2.3",
-		"verror": "1.10.0"
+		"assert-plus": "^1.0.0",
+		"extsprintf": "^1.3.0",
+		"json-schema": "^0.4.0",
+		"verror": "^1.10.0"
```

Comment:
Why are you adding carets? If you think this is a good idea, you should separate this into a different PR, otherwise you risk that this PR will get lost in discussion due to that unexpected change. Or was that merely by mistake?

Reply:
Got it from the previous PR.